modules/test/playwright/tests/frontend-data-set-web/main/tests/advanced/search.spec.ts

```diff
@@ -128,5 +127,46 @@ test(
 				).not.toBeVisible();
 			});
 		});
+
+		await test.step('Applying the same search twice does not show "Requesting Results for:" indefinitely', async () => {
+			await test.step('Reset the tab', async () => {
+				await fdsSamplePage.selectTab('Advanced');
+
+				await waitForFDS({
+					page,
+					visualizationMode: EFDSVisualizationMode.TABLE,
+				});
+			});
+
+			await test.step('Search for "Sample" and wait for the search to finish', async () => {
+				await fdsSamplePage.managementToolbar.searchInput.fill(
+					'Sample'
+				);
+
+				await fdsSamplePage.managementToolbar.container
+					.getByRole('button', {name: 'Search'})
+					.click();
```

Comment:
could we make this locator part of what `fdsSamplePage` provides, like we did with `searchInput` locator?